internal/manifests/manifests.go

```diff
@@ -15,12 +17,31 @@ import (
 	"github.com/os-observability/tempo-operator/internal/manifests/serviceaccount"
 )
 
+// Params holds parameters used to create Tempo objects.
+type Params struct {
```

Comment:
Can you use the `Params` struct from https://github.com/os-observability/tempo-operator/blob/4c03363347a3ae70f3272accd117fa7febaa8734/internal/manifests/manifestutils/params.go? It's identical, only moved to another package due to a circular dependency.